### ISSUE TITLE
Consensus Error with Grandpa

### ISSUE THREAD

**Issue by freehere107** [darwinia-network/darwinia#103]
block stuck at 1000

```
7|darwinia | 2019-11-22 06:03 +00:00: 2019-11-22 06:03:24 Encountered consensus error: ClientImport("Execution(Wasmi(Trap(Trap { kind: Unreachable })))")
7|darwinia | 2019-11-22 06:03 +00:00: 2019-11-22 06:03:27 Starting consensus session on top of parent 0x6bd22ebc9daf5385a353137ea65e454ba7e97a14db3b68768840a6306e71d588
```

**Comment by HackFisher** [comment]
Related issue: https://github.com/paritytech/substrate/issues/2834

**Comment by AurevoirXavier** [comment]
Fixed: #133.
